CONTRIBUTING.md

```diff
@@ -4,71 +4,77 @@ Please note that this project is released with a [Contributor Code of Conduct](h
+If it's a **tutorial or a blog post**, they get outdated really quickly so we don't really put them on the list but if it is on a very advanced and/or specific topic, we will consider it!
+If something is awesome, share it (pull request or issue or chat), let us know why and we will help you!
 
-- **To add to the list:** Submit a pull request or open an issue
-- **To remove from the list:** Submit a pull request or open an issue
-- want to change something: Submit a pull request or open an issue
-- want to try something else: Submit a pull request or open an issue
-- don't know what to do: Submit a pull request or open an issue, let us know what's going on.
+# Quality Standards
 
-or you can
+Note that we can help you achieve those standards, just try your best and be brave.
+We'll guide you to the best of our abilities.
 
-#### Join the chat:
+To be on the list, it would be *nice* if entries adhere to these quality standards:
 
-[![Join the chat at https://gitter.im/veggiemonk/awesome-docker](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/veggiemonk/awesome-docker)
+- Generally useful to the community.
+- A project on GitHub with a well documented `README.md` file and plenty of examples is considered high quality.
+- Clearly stating if an entry is related to (Linux) containers and not to Docker. There is an [awesome list](https://github.com/Friz-zy/awesome-linux-containers) for that.
+- Clearly stating "what is it" i.e. which category it belongs to.
+- Clearly stating "what is it for" i.e. mention a real problem it solves (even a small one). Make it clear for the next person.
+- If it is a **WIP** (work in progress, not safe for production), please mention it. (Remember the time before Docker 1.0 ? ;-) )
+- Always put the link to the GitHub project instead of the website!
 
-or you can
+To be on the list, the project **must** have:
 
-#### ping us on twitter:
+- How to setup the project
+- How to use the project (examples)
 
-* [veggiemonk](https://twitter.com/veggiemonk)
-* [idomyowntricks](https://twitter.com/idomyowntricks)
-* [gesellix](https://twitter.com/gesellix)
-* [dmitrytokarev](https://twitter.com/dmitrytok)
 
-`awesome-docker` is a hand-crafted list for high-quality information about docker and its resources.
-It would be nice if it is related or compatible with docker in some way.
-But if it's awesome, let's us know why and you're good to go!
+If your PR is not merged, we will tell you why so that you may be able to improve it.
+But usually, we are pretty relaxed people, so just come and say hi, we'll figure something out together.
+
+# Contribution Guidelines
+
+## I want to share a project, what should I do?
+
+- **Adding to the list:** Submit a pull request or open an issue
+- **Removing from the list:** Submit a pull request or open an issue
+- Changing something else: Submit a pull request or open an issue
+- Don't know what to do: Open an issue or join our chat, let us know what's going on.
```

Comment:
Thanks for the hint @gesellix 
Originally it was there right after this text.
Just updated.